_posts/en/newsletters/2024-03-20-newsletter.md

```diff
@@ -0,0 +1,160 @@
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [Bitcoin Core 26.1rc1][] is a release candidate for a maintenance release
+  of the network's predominant full node implementation.
+
+- [Bitcoin Core 27.0rc1][] is a release candidate for the next major
+  version of the network's predominant full node implementation.
+
+<!-- FIXME:harding to update on Tuesday -->
+
+## Notable code and documentation changes
+
+_Notable recent changes in [Bitcoin Core][bitcoin core repo], [Core
+Lightning][core lightning repo], [Eclair][eclair repo], [LDK][ldk repo],
+[LND][lnd repo], [libsecp256k1][libsecp256k1 repo], [Hardware Wallet
+Interface (HWI)][hwi repo], [Rust Bitcoin][rust bitcoin repo], [BTCPay
+Server][btcpay server repo], [BDK][bdk repo], [Bitcoin Improvement
+Proposals (BIPs)][bips repo], [Lightning BOLTs][bolts repo],
+[Bitcoin Inquisition][bitcoin inquisition repo], and [BINANAs][binana
+repo]._
+
+*Note: the commits to Bitcoin Core mentioned below apply to its master
+development branch and so those changes will likely not be released
+until about six months after the release of the upcoming version 27.*
+
+- [Bitcoin Core #27375][] adds support to the `-proxy` and `-onion`
+  feature for using Unix domain sockets rather than local TCP ports.
+  Sockets can be faster than TCP ports and offer different security
+  tradeoffs.
+
+- [Bitcoin Core #27114][] allows adding "in" and "out" to the
+  `whitelist` configuration parameter to give special access to
+  particular _incoming_ and _outgoing_ connections.  By default, a peer
+  listed in the whitelist will only receive special access when it
+  connects to the user's local node (an incoming connection).  By
+  specifying "out", the user can now ensure a peer receives special
+  access if the local node connects to it, such as by the user calling
+  the `addnode` RPC.
+
+- [Bitcoin Core #29306][] adds [sibling eviction][topic sibling
+  eviction] for transactions descended from an unconfirmed [v3
+  parent][topic v3 transaction relay].  This can provide a satisfactory
+  alternative to [CPFP carve-out][topic cpfp carve out], which is
+  currently used by [LN anchor outputs][topic anchor outputs].  V3
+  transaction relay is not currently enabled for mainnet and (as noted
+  above) this PR is merged into the development branch that is not
+  expected to be released for about six months.
+
+- [LND #8310][] allows the `rpcuser` and `rpcpass` (password)
+  configuration parameters to be retrieved from the Unix environment.
+  This can allow, for example, a `lnd.conf` file to be managed using a
+  non-private revision control system without storing the private
+  username and password.
+
+- [Rust Bitcoin #2458][] adds support for signing [PSBTs][topic psbt]
+  for transactions that include taproot inputs.
+
+{% assign day_after_posting = page.date | date: "%s" | plus: 86400 | date: "%Y-%m-%d 14:00" %}
```

Comment:
```suggestion
{% assign day_after_posting = page.date | date: "%s" | plus: 86400 | date: "%Y-%m-%d 14:30" %}
```

We had yet another scheduling change as we realized the irc meeting conflict
cc: @murchandamus